proto/storage/runtime_filters.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package storage;
+
+option go_package = "./;storage";
+option java_package = "io.stackrox.proto.storage";
+
+message RuntimeFilter {
+  enum RuntimeFilterFeatures {
+    EXTERNAL_IPS = 0;
+    PROCESSES = 1;
+    NETWORK_CONNECTIONS = 2;
+    LISTENING_ENDPOINTS = 3;
+  }
+
+  message RuntimeFilterRule {
+    string resource_collection_id = 1;
+    string status = 2;
```

Comment:
AIUI, `status` would be a kind of action specific to the Feature; mostly turning it on/off, but potentially something else, like limiting the arguments for PROCESSES ?

Reply:
Status is now an object.